packages/ibm-products/src/components/Datagrid/_storybook-styles.scss

```diff
@@ -114,3 +114,11 @@ $block-class: #{c4p-settings.$pkg-prefix}--datagrid;
   margin-right: $spacing-03;
   background-color: $field-01;
 }
+
+.#{c4p-settings.$carbon-prefix}--body--with-modal-open {
+  .#{$block-class}__mobile-toolbar-modal.cds--modal {
```

Comment:
```suggestion
  .#{$block-class}__mobile-toolbar-modal.#{c4p-settings.$carbon-prefix}--modal {
```

Reply:
Updated as requested and V1 PR #3069 raised